src/users.js

```diff
@@ -1,20 +1,26 @@
 import { ME_USER_NAME_PLACEHOLDER } from 'apify-shared/consts';
 import { checkParamOrThrow, pluckData } from './utils';
 
+/**
+ * Users
+ * @memberOf ApifyClient
+ * @namespace users
+ */
+
 export const BASE_PATH = '/v2/users';
 
 export default {
     /**
-     * Returns public profile of user for given user ID or username.
-     * If user ID is not specified it returns info about your account.
+     * Returns private and public information about user account.
      *
      * @memberof ApifyClient.users
      * @instance
      * @param {Object} options
-     * @param {String} [options.token]
-     * @param {String} [options.userId = 'me'] - User ID or username of the user
-     * @param callback
-     * @returns {UserInfo}<
+     * @param {String} [options.token] - If is set, it shows private and public information about user account, otherwise only public.
```

Comment:
"If set, the function returns a private and public information for the user account, otherwise it only returns public information."